plugins/by-name/luasnip/default.nix

```diff
@@ -104,31 +309,30 @@ in
           require("luasnip.loaders.from_vscode").lazy_load({['paths'] = {'/nix/store/.../path/to/snippets'}})
         ```
       '';
-      type = types.listOf loaderSubmodule;
     };
 
-    fromSnipmate = mkOption {
+    from_snipmate = mkOption {
```

Comment:
Similar to https://github.com/nix-community/nixvim/pull/2653#discussion_r1885069208 we want extraOptions to be `camelCase`